package.json

```diff
@@ -20,10 +20,14 @@
   "types": "./dist/typescript-library-starter.d.ts",
   "scripts": {
     "docs": "api-extractor run --typescript-compiler-folder ./node_modules/typescript && api-documenter markdown -i ./temp -o ./docs",
-    "lint": "eslint .",
+    "lint": "npm-run-all lint:*",
+    "lint:eslint": "eslint .",
+    "lint:tsc-src": "tsc --noEmit",
```

Comment:
This typecheck is redundant on CI because it is typechecked during the build during yarn prepare which is run after yarn install.

Reply:
Hmm, that is is only true on the first clone not in general, right? As in, when running local, if I run `yarn lint` it should check all the files and include testing the tests and src.